libraries/ESP8266HTTPClient/examples/BasicHttpClient/BasicHttpClient.ino

```diff
@@ -40,34 +42,39 @@ void loop() {
   // wait for WiFi connection
   if ((WiFiMulti.run() == WL_CONNECTED)) {
 
+    WiFiClient client;
+
     HTTPClient http;
 
     USE_SERIAL.print("[HTTP] begin...\n");
     // configure traged server and url
     //http.begin("https://192.168.1.12/test.html", "7a 9c f4 db 40 d3 62 5a 6e 21 bc 5c cc 66 c8 3e a1 45 59 38"); //HTTPS
-    http.begin("http://192.168.1.12/test.html"); //HTTP
+    if (http.begin((Client &)client, "http://tls.mbed.org/")) {  // HTTP
```

Comment:
Any reason this one picked?  Isn't tls.mbed.org https-only?

Reply:
Also, thanks for checking return codes!

Reply:
I will change this in http://jigsaw.w3.org/HTTP/connection.html